internal/cmd/version.go

```diff
@@ -28,7 +28,7 @@ func version(cli *CLI) error {
 	defer w.Flush()
 
 	fmt.Fprintln(w)
-	fmt.Fprintln(w, "Client:\t", strings.TrimPrefix(internal.Version, "v"))
+	fmt.Fprintln(w, "Client:\t", strings.TrimPrefix(api.GetClientVersion(), "v"))
```

Comment:
I think we want to keep using `internal.Version` here. 

I think the user is more interested in "what version did this Go binary come from", not "what version of the API is it using".

The API version is misleading to a user because the CLI can use endpoints that did not exist in that version.

Reply:
You're completely right. This will be reverted in #1972